presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
```

Comment:
nit `, which, when enabled,` reads a bit clunky .
Can we simplify the entire sentence.